ui/src/app/pages/organization/organization.component.ts

```diff
@@ -21,11 +21,10 @@ import { OrganizationService } from './organization.service';
   selector: 'app-organization',
```

Comment:
looks like this component can be removed

Reply:
there will be a separate task for organization module deleting